doc/user_guide/recordflux_ug/01-about_this_guide.rst

```diff
@@ -0,0 +1,94 @@
+.. _About_This_Guide:
+
+About This Guide
+~~~~~~~~~~~~~~~~
+
+This guide describes the use of RecordFlux,
+a toolsuite for specifying binary messages used in
+communication protocols and producing formally verifiable
+parsers and message generators. You specify messages in a
```

Comment:
We should add state machines similar to the README and the GitHub description where we say: "Formal specification and generation of verifiable binary parsers, message generators and protocol state machines "